### PR TITLE
Fix Mac whitespace using astyle

### PR DESCRIPTION
**Pull request by philburk**
EXPERIMENT - This is just a test to see if astyle
can be used to cleanup whitespace without making
other unwanted changes.

This was implemented using:

    astyle src/hostapi/coreaudio/*.c
    astyle src/hostapi/coreaudio/*.h

### CODE REVIEW

File: src/hostapi/coreaudio/pa_mac_core.c [hunk 1]
```diff
@@ -89,124 +89,124 @@ PaError PaMacCore_Initialize( PaUtilHostApiRepresentation **hostApi, PaHostApiIn
...
+    if( err != paNoError )
+        return NULL;
+    PaMacAUHAL *macCoreHostApi = (PaMacAUHAL*)hostApi;
+    AudioDeviceID hostApiDevice = macCoreHostApi->devIds[device];
+    CFStringRef nameRef;
+
+    /* First try with CFString */
+    UInt32 size = sizeof(nameRef);
+    error = PaMacCore_AudioDeviceGetProperty( hostApiDevice,
+            channelIndex + 1,
```
Comment (philburk) [inline src/hostapi/coreaudio/pa_mac_core.c:142]: This is not great but it was worse before.

File: src/hostapi/coreaudio/pa_mac_core.c [hunk 2]
```diff
@@ -278,11 +278,11 @@ static PaError IsStreamStopped( PaStream *s );
 static PaError IsStreamActive( PaStream *stream );
 static PaTime GetStreamTime( PaStream *stream );
 static OSStatus AudioIOProc( void *inRefCon,
-                               AudioUnitRenderActionFlags *ioActionFlags,
-                               const AudioTimeStamp *inTimeStamp,
-                               UInt32 inBusNumber,
-                               UInt32 inNumberFrames,
-                               AudioBufferList *ioData );
+                             AudioUnitRenderActionFlags *ioActionFlags,
```
Comment (philburk) [inline src/hostapi/coreaudio/pa_mac_core.c:281]: This continuation looks good.

**Review by philburk — COMMENTED** [review]
astyle seems to do a good job with basic indentation.
It does not mess with other formatting.
But continuation lines are a not ideal.
It may be useful if combined with a little manual formatting.

**Comment by MichalPetryka** [comment]
I've used my tools on the files to check if astyle fixed everything, looks like only 2 files have doubled newlines at the end, everything else seems correct https://github.com/MichalPetryka/portaudio/commit/24e06fe6c64309e9f0d9e4206cad8faff02694a4

**Comment by RossBencina** [comment]
Superseded by #502

**Comment by RossBencina** [comment]
@philburk this can be closed now.

**Pull request closed without merging**